src/components/NotFoundPage.tsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from 'react';
+import { LangContext } from '../utils/LangContext';
+import { getTranslation } from '../utils/getTranslation';
+import { Typography } from 'antd';
+import MainPic from '../images/film.png';
+
+export const NotFoundPage = () => {
+  const lang = useContext(LangContext);
+  const { Title } = Typography;
+
```

Comment:
`const { Title } = Typography;` should be moved out of component
```suggestion
import MainPic from '../images/film.png';

const { Title } = Typography;
  
export const NotFoundPage = () => {
  const lang = useContext(LangContext);

```